examples/sdxl.py

```diff
@@ -303,7 +303,7 @@ def __call__(self, denoiser, x:Tensor, s:Tensor, c:Dict, uc:Dict) -> Tensor:
 
 class SplitVanillaCFG(Guider):
   def __call__(self, denoiser, x:Tensor, s:Tensor, c:Dict, uc:Dict) -> Tensor:
-    x_u = denoiser(x, s, uc)
+    x_u = denoiser(x, s, uc).clone().realize()
```

Comment:
do you need the realize or just the clone?

Reply:
you do have to realize, since if you don't then the graph references the original buffer for the JIT and it gets overwritten, the realize is what actually copies the data into a new buffer